EphemeralElixir/EphemeralElixir
# Feat move markers

#### What does this PR do?

Add 4 fixed positions for integration testing.
#### Where should the reviewer start?

Gmap.jsx
#### How should this be manually tested?

Login up to 4 users and you'll see their markers in 4 different locations on the map.
#### Any background context you want to provide?
#### What are the relevant tickets on Pivotal Tracker?
#### Provide screenshots (if appropriate).

<img width="771" alt="screen shot 2016-06-02 at 2 29 14 pm" src="https://cloud.githubusercontent.com/assets/770322/15761714/91d2bc8c-28ce-11e6-84c5-4accbe34c369.png">
